src/main/java/CommandScanner.java

```diff
@@ -0,0 +1,37 @@
+public class CommandScanner {
+    public enum Commands {TODO, DEADLINE, EVENT, LIST, MARK, UNMARK, DELETE, BYE}
+
+    public static String scanCommand(String[] inputs) throws DukeException {
+
+        if (inputs.length > 0) {
+            String firstWord = inputs[0].toUpperCase();
```

Comment:
Maybe we can use a name like inputCmd instead of firstWord to make it more specific?